webpack.config.js

```diff
@@ -0,0 +1,158 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerWebpackPlugin = require('css-minimizer-webpack-plugin');
+const TerserWebpackPlugin = require('terser-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
+
+const isDev = process.env.NODE_ENV === 'development';
+const isProd = !isDev;
```

Comment:
This can be true, but it's not always like that, so consider doing more verbose, but safer check:
```suggestion
const isProd = process.env.NODE_ENV === 'production'
```